### PR TITLE
Small refactoring

### PR DESCRIPTION
**Pull request by edumorlom**
Some small code refactoring + adds package.json.

### CODE REVIEW

File: covid19-dashboard/src/Utils.ts [hunk 1]
```diff
@@ -26,14 +26,20 @@ export const numberWithCommas = (num: number): string => {
 };
 
 /**
- * Gets the latest date in the for a a list of Iso-dates.
+ * Gets the min or max date in a list of ISO dates.
```
Comment (edumorlom) [inline covid19-dashboard/src/Utils.ts:29]: Revert back to getLatestDate

Comment (edumorlom) [reply covid19-dashboard/src/Utils.ts:29]: Can we do this in a separate PR so that can rename the function in the necessary files?

Comment (beets) [reply covid19-dashboard/src/Utils.ts:29]: perhaps this PR isn't complete then. I don't see the renamed function used in these files (or it's not obvious anyway). Could you just revert this change?

**Comment by edumorlom** [comment]
I reverted the getMinMaxDate to getLatestDate.
Other changes remain

**Review by beets — APPROVED** [review]

**Pull request merged**